next.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  // TODO: remove eslint ignore until all sources are properly linted
+  // https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint
   eslint: {
-    ignoreDuringBuilds: true
+    ignoreDuringBuilds: false
```

Comment:
I thought this is `true` because of we already have GH action for lint #29

Reply:
Ah you're right, let me revert that. 🙏🏻

Reply:
Reverted on 59ee8031404bc2c24e0fc167476875d760c9cb35 with comments describing reason disabling ESLint on build.